### ISSUE TITLE
Nostroots Collective 2024-05

### ISSUE THREAD

**Issue by chmac** [Trustroots/nostroots#11]
tl;dr We’re organizing a Nostr Trustroots Collective this May near Berlin.

## What

We want to nostrify trustroots.

## When

Wednesday 22nd to Sunday 26th of May 2024.
Feel free to arrive a day or three beforehand.

## Where

1 hour train ride from Berlin. Precise details shared with participants.

## Why

In 2022 the main person behind Trustroots stepped away from the project, without even a message. When we realized email had been down for 5 weeks, we fixed it and made sure to increase the bus factor a little.

We think it is worth our time and energy to work towards gift economy social networks that do not rely on any specific person or organization, so effectively we want to make ourselves redundant. By moving onto Nostr development will become distributed and not centralized like it is now, so everyone can volunteer in their own autonomous way and with whomever they please.

Read more at https://github.com/Trustroots/nostroots

## How, goals

Implement basic features in Trustroots that will make it possible for anyone to build Nostr apps connecting this.

- run a trustroots relay server or two
- [only accessible to trustroots users](https://github.com/Trustroots/nostroots/issues/9)
- [utilize geo features](https://github.com/Trustroots/nostroots/issues/10)
- trust on nostr, any related NIPs?
- build adjacent nostr stuff, e.g. for hitchhikers or other [trustroots circles](https://www.trustroots.org/circles)
   * [nostr notes relevant to hitchhikers](https://github.com/Hitchwiki/nostrides/issues/1), also great for trustroots   hitchhiking circle and hitchwiki
   * [NIP-07 log in to mediawiki extension](https://github.com/Hitchwiki/nostrides/issues/2), great for hitchwiki and trashwiki

Wanna come join? Fill in [this form](https://formstr.app/#/fill/a83844ae142eac2e6347b172040b75dbf8fa6fa810a30894ab2bf2e8c58cf92d).

(If you can't make it or want to participate remotely you can use [this Google Form](https://docs.google.com/forms/d/1Fp24fntRz0ILxVsVpLaA3t0DI2H9sWYpX63fFXDYoro/viewform?edit_requested=true))

**NOTE**: We are not seeking general feedback on this nostr mission, but if you insist on sharing that, please do so in a new issue and not here. This issue is about organising the collective.

**Comment by Thatoo** [comment]
May I express my concern to see Trustroots becoming like Hospitalityclub and Couchsurfing an isolated plateform keeping its users captive...
Please reconsider this project and rejoin the effort of an Open Hospitality Network through the fediverse.

**Comment by guaka** [comment]
@Thatoo your concern is welcome. But in this case it seems quite a bit out of place, considering that what we're trying to achieve is exactly the opposite of keeping users captive. Nostr is much less captive than user/pass auth based fediverse nodes.

Anyway, you are welcome to work on whatever makes users less captive, here are some ideas: https://github.com/Trustroots/trustroots/issues/2669
And if we manage to progress with nostroots it can make sense to look into https://gitlab.com/soapbox-pub/mostr to set up bridges to the hospex fediverse.

**Comment by guaka** [comment]
OK, found this issue https://github.com/BeWelcome/rox/issues/90, no actual ActivityPub work done, and well, let me put it this way, it's a bit lonely on the mount everest.

**Comment by chmac** [comment]
@Thatoo Please open a new issue for discussion or feedback about the project in general, this issue is about the collective in May.

**Comment by rosano** [comment]
I'm a bit sad the browser web interface is unsustainable, as many people might not be able to easily navigate these more tech-leaning concepts and spaces. This discussion is also on GitHub which generally favours people with a developer mindset.

I tried to learn a bit about nostr to understand what this would mean and it isn't clear yet to me but I'll keep exploring more; I read the [Wikipedia page](https://en.wikipedia.org/wiki/Nostr), this [introduction](https://wiki.wellorder.net/post/nostr-intro/), and the [comparison with Mastodon / ActivityPub](https://nostr.com/comparisons/mastodon). Might be good to share with any announcements some inclusively-written introductions to the technology and how Trustroots would work there (maybe I could even make one later).

**Comment by zweifeln** [comment]
I hate to do this.
I love open source. I love the sharing economy. (Though I don't really believe in the "gift" economy. Gift and economy doesn't go together. But that's beside the point.)

Every de-central approach; every maintained open source project has someone (or a rather small group of people) who take care of it. Sometimes they have a business model (like my employer does), sometimes these people have a different form of income and do it as their hobby. That's great. Sometimes these people will stop maintaining things and then half the internet may break. That's bad but it is how it has been the past 30 years.
I strongly believe it is very helpful to acknowledge that before going forward with any decentralizing idea.

Someone is actually in charge.
Someone needs to write the code and maintain it. Install updates and keep the servers running. Yes, software can update itself - but only as long as the updating part of the software itself works all fine. Software the harddrives need to be changed and sometimes software just breaks. Then there is need for someone who knows what's happening. So we *rely* on them.
That is the case no matter how good a programmer one might be. If you didn't work with that software, you don't know the libraries and frameworks.
We're having this topic here on Github. A platform owned (and maintained) by microsoft.
The only really free, decentralized software widely used out there … is e-mail. But who of you actually runs their own e-mail server? (I don't. It's too much pain.)
The point cannot be to be totally independent. Yes, that sounds nice but in reality it means, you are alone.
The point is *trust*. The thing that is already in our name. Some people have the knowledge and willpower to develop software and run it. Others will trust them and use it. That is okay. And that means, the people not running the software will trust their platform. Or not - then they will leave the platform.
Similar to how not every human on earth is making their own bread. Most of us are just buying it in a bakery. (Or don't eat bread). It is not sustainable for everyone to make their own bread. And certainly not practicable for everyone to grow their own grain.

We rely on people not on software. That is a good thing. Let's acknowledge that.
When we do that, we can go forward and make some system that is easier to maintain and better usable to more people - also non tech-people.
Running decentralized software is - unfortunately - not actually decentralized. No matter how easy it seems to tech people. It may seem easy for us who we are comfortable using Github. But it actually is quite elite. Most people I have met traveling the past 8 weeks don't know what Github is.

The point is not the software. The point is how to make it sustainable. How to have a community that is trusting each other and how that spirit might continue even when central people step back for whatever reasons. There are lots of good reasons. So often people get pressured into keeping a central role as it all would fall apart if they step back. But maybe they got sick or take care of sick people or children or maybe they are just tired after doing it for so long and actually just wanna travel themselves and enjoy live.
The the hard question is: how to keep things running. How to make sure the community has enough trusted people to step in if someone steps back. How to develop and maintain a beautiful community.
That is more a philosophical than a technical question but that is the central question.

**Comment by chmac** [comment]
@zweifeln I moved your comment to a new issue, this issue is not the place for discussion of nostroots, it is for organising the collective.

**Comment by dreirik** [comment]
I just saw this event announcement by chance and wanted to ask if I can put you it / you want to put it on www.foss.events ?

**Comment by chmac** [comment]
> I just saw this event announcement by chance and wanted to ask if I can put you it / you want to put it on www.foss.events ?

I added it now.

**Comment by dreirik** [comment]
> > I just saw this event announcement by chance and wanted to ask if I can put you it / you want to put it on [www.foss.events](http://www.foss.events) ?
> 
> I added it now.

awesome, I just approved it. If you have more details (e.g. about the location) you can always come back and edit the event on the bottom of its website: https://foss.events/2024/05-22-nostroots-collective.html

**Comment by guaka** [comment]
collective landing page at https://collective.trustroots.org/

**Comment by guaka** [comment]
it has been glorious!
